packages/devtools_app/lib/src/app_size/file_import_container.dart

```diff
@@ -35,6 +38,9 @@ class FileImportContainer extends StatefulWidget {
 
   final void Function(String error) onError;
 
+  // The file's extensions where we are going to get the data from.
```

Comment:
nit: use /// for dart doc. And maybe "the valid file extensions for this file selector"?